src/main/java/me/zeroeightsix/kami/module/modules/render/AntiFog.kt

```diff
@@ -1,17 +1,18 @@
 package me.zeroeightsix.kami.module.modules.render
 
+import me.zeroeightsix.kami.util.KamiLang 
 import me.zeroeightsix.kami.module.Module
 import me.zeroeightsix.kami.setting.ModuleConfig.setting
 
 /**
  * Created by 086 on 9/04/2018.
  */
 object AntiFog : Module(
-    name = "AntiFog",
-    description = "Disables or reduces fog",
+    name = KamiLang.get("module.modules.render.AntiFog.Antifog"),
+    description = KamiLang.get("module.modules.render.AntiFog.DisablesOrReducesFog"),
     category = Category.RENDER
 ) {
-    val mode = setting("Mode", VisionMode.NO_FOG)
+    val mode = setting(KamiLang.get("module.modules.render.AntiFog.Mode"), VisionMode.NO_FOG)
```

Comment:
Use extension function and get the class package of `this`

Reply:
???

Reply:
Instead of doing `KamiLang.get("module.modules.whatever.ModuleName.Whatever")` every time, it would be easier to do `this.il8n.get("Mode")`

Reply:
ok, thanks.